src/componentes/Cadastro/Cadastro.jsx

```
@@ -1,200 +1,207 @@
+				/>
+				<span className={style.CadastroCampoWP}>
+					<input
+						className={style.CadastroCampoCheck}
+						type="checkbox"
+						value={whatsapp}
+						onChange={() => {
+							setWhatsapp(!whatsapp);
+						}}
+					/>
+					<label> Esse número possui WhatsApp </label>
+				</span>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)